pkg/tsdb/grafana-pyroscope-datasource/pyroscopeClient.go

```diff
@@ -115,7 +114,6 @@ func (c *PyroscopeClient) GetSeries(ctx context.Context, profileTypeID string, l
 
 	resp, err := c.connectClient.SelectSeries(ctx, req)
 	if err != nil {
-		logger.Error("Received error from client", "error", err, "function", logEntrypoint())
 		span.RecordError(err)
 		span.SetStatus(codes.Error, err.Error())
 		return nil, backend.DownstreamError(fmt.Errorf("received error from client while getting series: %w", err))
```

Comment:
uber nit: I think that these all can be `backend.DownstreamErrorf("received error from client while getting series: %w", err)`